### PR TITLE
Scotland content

### PR DESCRIPTION
**Pull request by etheya**
New content for scottish free hours.

### CODE REVIEW

**Comment by etheya** [comment]
HAs someone just added new content in for Wales?

**Comment by sara202** [comment]
"HAs someone just added new content in for Wales?" - does this affect your work??

**Comment by etheya** [comment]
I have changed the content for scotland - new ask from Donna this afternoon, didnt touch wales but couldnt see if anyone had or not. Thats all.

File: conf/messages.en [hunk 1]
```diff
@@ -110,15 +110,15 @@ location.error = Select where you live
 childAgedTwo.title = Do you have any children aged 2?
 childAgedTwo.heading = Do you have any children aged 2?
 childAgedTwo.guidance.england = Some 2 year olds in England could be entitled to 15 hours free childcare a week in term time – this is 570 hours a year.
-childAgedTwo.guidance.wales = Some 2 year olds in Wales could be entitled to a minimum of 10 hours of funded early education a week in term time – this is 390 hours a year.
```
Comment (sara202) [inline conf/messages.en]: This guidance changed here is for Wales, not Scotland - is this right?

Comment (sara202) [reply conf/messages.en]: I have spoken to Donna and Stu and resolved this.

**Pull request merged**